src/NuGet.Core/NuGet.PackageManagement/PublicAPI.Unshipped.txt

```diff
@@ -1,3 +1,4 @@
+NuGet.ProjectManagement.BuildIntegratedPackageReference.BuildIntegratedPackageReference(NuGet.LibraryModel.LibraryDependency dependency, NuGet.Frameworks.NuGetFramework projectFramework, NuGet.Packaging.Core.PackageIdentity installedVersion) -> void
```

Comment:
Not requesting any change, just showing my sadness that VS UI changes are affecting the NuGet SDK 😞   It shows a "separation of concerns" failure in the past, probably due to leaky abstractions, or not even attempting to abstract lower level types.